client/index.html

```diff
@@ -265,6 +265,38 @@ <h1 class="title">Settings</h1>
 									Enable notification for all messages
 								</label>
 							</div>
+							<% if (!public) { %>
+							<div id="change-password">
+								<form action="" method="post">
+									<div class="col-sm-12">
+										<h2>Change password</h2>
+									</div>
+									<div class="col-sm-12">
+										<label aria-label="Enter current password">
+											<span class="sr-only">Enter current password</span>
+										</label>
+										<input class="input" type="password" name="old_password" placeholder="Enter current password">
```

Comment:
This is not exactly how to use use `label` and assistive technology attributes like `sr-only` or `aria-label`:
- A `label` must be tied to its `input` with `for`
- The `aria-label` must be on the `input` instead of the `label`

Additionally, `sr-only` and `aria-label` serve the same purpose, so only one of them is enough. Remove the `label`s, add `aria-label="Enter current password"` to the `input`s and that should do :-)